internal/commands/push/command.go

```diff
@@ -198,7 +198,35 @@ func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.C
 		if err := clients.Realm.ImportDependencies(appRemote.GroupID, appRemote.AppID, uploadPathDependencies); err != nil {
 			return err
 		}
-		ui.Print(terminal.NewTextLog("Uploaded dependencies archive"))
+
+		ui.Print(terminal.NewTextLog("Uploaded dependencies"))
+		s := spinner.New(terminal.SpinnerCircles, 250*time.Millisecond)
+		s.Suffix = " Installing dependencies: starting..."
+
+		s.Start()
```

Comment:
these start/stop calls should be inside the `waitForInstallation` function scope (to ensure the `defer` behaves correctly)